doc/README.md

```diff
@@ -163,4 +163,4 @@ In the `lwjgl3` repository:
 In the `lwjgl3-www` repository:
 
 - `client/routes/customize/BuildConfigurator/lwjgl/nightly.js` (`lwjgl3-www` repository)
-    * Add definition for the binding.
\ No newline at end of file
+    * Add definition for the binding.
```

Comment:
I'm curious: What happened in this line?

Reply:
A newline `\n` was added to the end of the file. It is a convention in git to end all files with a newline for better diffs when appending to the end of a file.